packages/ladle/lib/cli/vite-base.js

```diff
@@ -60,6 +60,10 @@ const getBaseViteConfig = async (ladleConfig, configFolder, viteConfig) => {
   debug("User provided @vite/plugin-react: %s", hasReactPlugin);
   debug("User provided @vite/plugin-react-swc: %s", hasReactSwcPlugin);
 
+  const hasDisabledReactPlugin = ladleConfig?.disableReactPlugin;
```

Comment:
are we missing the part where we use this setting to actually disable the plugin?

Reply:
@Scott-Fischer ? I would love to see this PR finished 🙏

Reply:
@tajo Yeah it was missing! 🤦 

Thanks for the catch.